src/pages/policy-details/policy-details.tsx

```diff
@@ -28,35 +32,73 @@ export default function PolicyDetails() {
   if (!policy) {
     return (
       <BaseLayout subtitle={`Policy ${policyId}`}>
-        <h4 className={styles.heading}>Policy {abbrStr(policyId)}</h4>
+        <h4 className={styles.heading}>Policy</h4>
```

Comment:
We are not showing the policyId anymore, right? (Just double checking)

Reply:
👍 We show the `policy.metadata`

Reply:
Yeah, but in this field we show just a generic "Policy" so I meant that specifically.